s3-other.tf

```diff
@@ -96,7 +96,7 @@ resource "aws_s3_bucket" "apiary_managed_logs_bucket" {
   count  = local.enable_apiary_s3_log_management ? 1 : 0
   bucket = local.apiary_s3_logs_bucket
   acl    = "log-delivery-write"
-  tags   = merge(tomap({"Name"=local.apiary_s3_logs_bucket}), var.apiary_tags)
+  tags   = merge(tomap({"Name"=local.apiary_s3_logs_bucket, "AWS_S3_013"="DISABLE"}), var.apiary_tags)
```

Comment:
instead of put hardcoded tag value here, can we have an `extra_apiary_tags` Or `extra_apiary_system_tag` created to handle this case ?

Reply:
Also pls update change log

Reply:
I have pushed the comments, please check them and let me know if that was what you were expecting, thanks!